AirCasting/Graph/Graph.swift

```diff
@@ -105,6 +108,13 @@ class AirCastingGraph: UIView {
         }
     }
     
+    func updateMidnightLines(with limitLines: [ChartLimitLine]) {
+        lineChartView.xAxis.removeAllLimitLines()
```

Comment:
Maybe store the last displayed and only update on view if the values are different?